### PR TITLE
test(organisms): Add smoke test for touts

### PR DESCRIPTION
**Pull request by TatsianaMiraniuk**
## Changes
- Added smoke test for Touts
- Added a package to support the Xpath selectors
- Added a package to support uploading a file
- Set the default browser viewport sizes
- Added two custom commands

## Test Instructions
 - NOTE: due to a content loss issue (#14) the test may fail in Master

### CODE REVIEW

File: cypress/integration/smokeTest/touts_spec.js [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+
+  it('touts: 8 - checking that the data still present in Edit Mode after switching back from Preview Mode', () => {
+    cy.clickEdit()
+    cy.xpath(titleXpath)
+      .should('have.text', title)
+    cy.xpath(descriptionXpath)
+      .should('have.text', description);
+    cy.xpath(ctaLabelXpath)
+      .should('have.text', ctaLabel)
+    const fileName = 'images/img1.jpg'
```
Comment (asiarheyeu) [inline cypress/integration/smokeTest/touts_spec.js]: Can we move 'images/img1.jpg' and 'images/img2.jpg' to global constants? Here and across whole file.

File: cypress/integration/smokeTest/touts_spec.js [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+  const ctaLabelXpath = '//*[@id="tout-horizontal"]//*[@data-tout-element="link"]//div[@data-slate-editor="true"]'
+  const imagePlaceholderXpath = '//*[@id="tout-horizontal"]//img[@data-tout-element="image"]'
+  const urlFieldXpath = '//form[@aria-label="Context Menu Link Form"]//input[@id="link-href"]'
+  const linkIconXpath = '//*[@role="toolbar" and @aria-label="Local Context Menu"]//*[@aria-label="Link"]'
+  const checkmarkIconLinkFormXpath = '//form[@aria-label="Context Menu Link Form"]//button[@aria-label="Submit"]'
+  const altFieldXpath = '//form[@aria-label="Context Menu Image Form"]//input[@id="image-alt"]'
+  const checkmarkIconImageFormXpath = '//form[@aria-label="Context Menu Image Form"]//button[@aria-label="Submit"]'
+  const imageIconXpath = '//*[@role="toolbar" and @aria-label="Local Context Menu"]//*[@aria-label="Image"]'
+  const ctaButtonXpath = '//*[@id="tout-horizontal"]//a[@data-tout-element="link"]'
+  const imageLinkXpath = '//div[@id="tout-horizontal"]//a[@data-tout-element="image-link"]'
```
Comment (asiarheyeu) [inline cypress/integration/smokeTest/touts_spec.js]: Please order const declaration in asc order

Comment (TatsianaMiraniuk) [reply cypress/integration/smokeTest/touts_spec.js]: updated

File: cypress/integration/smokeTest/touts_spec.js (hunk 1 above)
Comment (TatsianaMiraniuk) [reply cypress/integration/smokeTest/touts_spec.js]: updated. renamed as agreed in Skype

File: cypress/integration/smokeTest/touts_spec.js (hunk 2 above)
Comment (TatsianaMiraniuk) [reply cypress/integration/smokeTest/touts_spec.js]: done

Comment (TatsianaMiraniuk) [reply cypress/integration/smokeTest/touts_spec.js]: reverted order of const to original version

**Pull request merged**